agent/client.go

```diff
@@ -126,6 +131,18 @@ func (r *client) LocalStatus(ctx context.Context) (*pb.NodeStatus, error) {
 	return resp.Status, nil
 }
 
+// Time returns the current time on the target node.
+func (r *client) Time(ctx context.Context, req *pb.TimeRequest) (time *pb.TimeResponse, err error) {
```

Comment:
Nit: I don't think named return arguments are really needed here.

Reply:
makes sense, still helps with readability so I'd keep 'em. But thanks for the nit :+1: